pkg/meshctl/commands/demo/istio-multicluster/install/demo_istio_multicluster_install.go

```diff
@@ -0,0 +1,280 @@
+package install
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/gobuffalo/packr"
+	"github.com/solo-io/service-mesh-hub/codegen/helm"
+	"github.com/solo-io/service-mesh-hub/pkg/common/defaults"
+	"github.com/solo-io/service-mesh-hub/pkg/common/version"
+	"github.com/solo-io/service-mesh-hub/pkg/meshctl/install/smh"
+	"github.com/solo-io/service-mesh-hub/pkg/meshctl/registration"
+	"github.com/solo-io/skv2/pkg/multicluster/register"
+	"github.com/spf13/cobra"
+)
+
+func Command(ctx context.Context, masterCluster string, remoteCluster string) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "install",
```

Comment:
```suggestion
		Use:   "init",
```

No strong feeling about this

Reply:
i used 'init' at first, but the compiler doesn't like imports or functions called 'init'

Reply:
renamed